rust/cache/src/bin/cops-disk-cache-config-writer.rs

```diff
@@ -0,0 +1,12 @@
+use clap::Parser;
+
+use chroma_cache::{CacheConfig, FoyerCacheConfig};
+
+fn main() {
```

Comment:
nit: comment that this takes the default config and generates the yaml would be helpful